src/get-started/install/linux.md

```diff
@@ -28,11 +45,7 @@ your development environment must meet these minimum requirements:
   - `which`
   - `xz-utils`
   - `zip`
```

Comment:
**question:** Did you want to add links and install instructions as you did in [9081](https://github.com/flutter/website/pull/9081)?